db/migrate/20211117170147_set_tenant_id.rb

```diff
@@ -0,0 +1,17 @@
+class SetTenantId < ActiveRecord::Migration[5.0]
+  
+  def up
+    Account.where("provider = 1 AND master IS NULL").update_all(master: false)
```

Comment:
The account table is quite big and is always used, an `UPDATE` will block the table for write.
This is not good in production as it will make web requests longer, probably killing the unicorn workers by timeout.

Reply:
As mentionned in a previous comment you can look at a previous example:

https://github.com/3scale/porta/blob/master/lib/tasks/services.rake#L28-L31

Which will lead you to

https://github.com/3scale/porta/blob/1e77f3f0bd6471d0b98e1d09e6081b23a6eaa8f1/app/workers/create_default_proxy_worker.rb